src/components/charts/list/DiscoverCharts.tsx

```diff
@@ -241,25 +241,16 @@ function DiscoverChartList() {
                         handleEnvironmentChange={handleEnvironmentChange}
                         handleNameChange={handleNameChange}
                         discardValuesYamlChanges={discardValuesYamlChanges}
-                    /> : <> <ChartListHeader chartRepoList={state.chartRepos}
```

Comment:
ChartListHeader has been removed completely, is this expected?

Reply:
Removed only in the case of the empty state, In case of charts have this header below
<img width="1332" alt="Screenshot 2022-03-04 at 3 59 49 PM" src="https://user-images.githubusercontent.com/61836271/156746850-35ea9a0c-fbf8-41b2-ae95-611d642abec2.png">